Extensions/xkit_patches.js

```diff
@@ -223,6 +223,14 @@ XKit.tools.dump_config = function(){
 	}, {});
 };
 
+// http://stackoverflow.com/a/901144/2073440
+XKit.tools.getParameterByName = function(name){
+	name = name.replace(/[\[]/, "\\[").replace(/[\]]/, "\\]");
+	var regex = new RegExp("[\\?&]" + name + "=([^&#]*)"),
```

Comment:
good enough for me would be just replace line 223 with name = encodeURIComponent(name)